### PR TITLE
Sign up frontend

### PR DESCRIPTION
**Pull request by iryb**
See https://app.asana.com/0/1202525679663009/1202735993104741

### Sign up form
sign-up-form

### Select focused
sign-up-select-focused

### Select selected
sign-up-select-selected

### CODE REVIEW

File: frontend/src/components/common/select-field/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    legend {
+      display: none;
+    }
+  }
+
+  .error {
+    margin-top: 0;
+    color: var(--red);
+    font-size: 12px;
+    font-family: Lato, sans-serif;
```
Comment (RomanMahola) [inline frontend/src/components/common/select-field/styles.module.scss]: Add "Lato" to css-variables and use it

File: frontend/src/components/sign/components/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            id="location"
+            name="Location"
+            value={location}
+            onChange={handleSelectChange}
+            onClose={handleSelectClose}
+            required={false}
+          >
+            <MenuItem value="kyiv">Kyiv</MenuItem>
+            <MenuItem value="kharkiv">Kharkiv</MenuItem>
+            <MenuItem value="odesa">Odesa</MenuItem>
```
Comment (RomanMahola) [inline frontend/src/components/sign/components/sign-up-form/sign-up-form.tsx:70]: We need all cities, or use API for this.
Try to find smth

**Review by RomanMahola — DISMISSED** [review]
Just two small remarks. You can fix it if you wish. Anyway I will approve PR and we can mege it any time

File: frontend/src/components/sign/components/sign-up-form/sign-up-form.tsx (hunk 2 above)
Comment (iryb) [reply frontend/src/components/sign/components/sign-up-form/sign-up-form.tsx:70]: @RomanMahola Cities list will be displayed using API call to backend. We have Locations table in the database. But on backend this functionality is not developed yet. That's why I left it as hardcode.

**Review by RomanMahola — APPROVED** [review]

**Pull request merged**